tests/components/aladdin_connect/test_init.py

```diff
@@ -24,8 +24,7 @@ async def test_unload_entry(hass: HomeAssistant):
         return_value=True,
     ):
 
-        assert (await async_setup_component(hass, DOMAIN, entry)) is True
-
+        await async_setup_component(hass, DOMAIN, entry)
```

Comment:
I think we want to use `hass.config_entries.async_setup` to set up the integration.